### PR TITLE
Feature/ACTP-382/Add warning function to the report class

### PR DESCRIPTION
**Pull request by shpran**
Add warning function to the report class

Related to: [https://oat-sa.atlassian.net/browse/ACTP-382](https://oat-sa.atlassian.net/browse/ACTP-382)

### CODE REVIEW

File: common/report/class.Report.php [hunk 1]
```diff
@@ -107,6 +107,17 @@ public function __construct($type, $message = '', $data = null)
         $this->elements = [];
         $this->data = $data;
```
Comment (SashaTAO) [inline common/report/class.Report.php]: Could you please move `__construct()` method to be the first method in this class

Comment (shpran) [reply common/report/class.Report.php]: Done

**Comment by SashaTAO** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by zagovorichev** [comment]
I'm sorry, @shpran, I guess that this feature is redundant, success and errors are enough for the report (at least for this task)

**Comment by shpran** [comment]
Unnecessary feature.

**Pull request closed without merging**